src/api/view/html/public/auth.js

```diff
@@ -0,0 +1,49 @@
+import { initializeApp } from "https://www.gstatic.com/firebasejs/9.4.0/firebase-app.js";
+import { getAuth, signInWithEmailAndPassword } from "https://www.gstatic.com/firebasejs/9.4.0/firebase-auth.js";
+// Import the functions you need from the SDKs you need
+// import { initializeApp } from "firebase/app";
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyC1f5cioxoKT1E06TEumrPzGPIwlWeW-oc",
+  authDomain: "food-app-7cb62.firebaseapp.com",
+  projectId: "food-app-7cb62",
+  storageBucket: "food-app-7cb62.appspot.com",
+  messagingSenderId: "602483393246",
+  appId: "1:602483393246:web:785a926d8178a2739f784f",
+  measurementId: "G-4GEJQZL0R3"
+};
```

Comment:
This shouldn't be in the source code/version control

Reply:
So this part is essentially to show the creation of the accessToken. Basically, everything in the public folder is just for reference. The config itself can be shown to anyone. It doesn't contain sensitive information.